api/test/unit/oauth/oauth.test.js

```diff
@@ -0,0 +1,277 @@
+"use strict";
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+var request = require("supertest");
+var expect = require("chai").expect;
+var fs = require("fs");
+var path = require("path");
+var HttpStatus = require('http-status-codes');
+
+var logger = require("../../../lib/log/logger");
+var oauth;
+var nock = require("nock");
+var theAppId = "the-app-guid";
+var theUserId = "the-user-id";
+var req = {};
+describe("Oauth", function() {
+  this.timeout(10000 + 1000); //timeout is 10s and here wait 11s
```

Comment:
where the 10s timeout is defined?

Reply:
apiserver communicate with other components(scalingengine,metricscollector&scheduler), the timeout is 10s. It is defined in lib/utils/*Utils.js